src/index.ts

```diff
@@ -42,12 +41,12 @@ class Webhooks<
     options: EmitterWebhookEvent & { signature: string }
   ) => Promise<void>;
 
-  constructor(options?: Options<E, TTransformed>) {
+  constructor(options?: Options<E, TTransformed> & { secret: string }) {
     if (!options || !options.secret) {
       throw new Error("[@octokit/webhooks] options.secret required");
     }
 
-    const state: State = {
+    const state = {
```

Comment:
```suggestion
    const state: State & { secret: string }= {
```

This needs to be explicitly typed as it's an object

Reply:
I've applied this change although I confess I don't understand it: why do objects need to be explicitly typed?

Reply:
Answered in https://github.com/octokit/webhooks.js/pull/466#discussion_r575863299, thanks!